src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx

```diff
@@ -53,17 +53,26 @@ For users on the [New Relic One user model](/docs/accounts/original-accounts-bil
 
 ## Overview of user management concepts [#understand-concepts]
 
-To optimally use our more advanced user management features, it's important to first understand the concept of the "access grant." An access grant gives a group of users access to a specific role on a specific account. For a New Relic [organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure) that has many accounts, groups typically require more than one access grant because users in a group usually need access to multiple accounts and roles. The diagram below explains the elements that make up an access grant.
+If your organization has [Pro or Enterprise edition](https://newrelic.com/pricing), you can create and configure access grants. An access grant gives a group of users access to a specific role on a specific account. 
```

Comment:
Why not change access grants to manage users?

Reply:
it's a bit complicated. basically, there are two separate UIs: the 'Organizations and access' UI where ths stuff happens, and then the 'User management' UI, where the actual users are. So saying 'manage users' can be a bit confusing cause there are these two UIs.